app/controllers/ApplicationController.scala

```diff
@@ -296,6 +296,36 @@ trait ApplicationController extends BaseController with GatekeeperAuthWrapper {
     }
   }
 
+  def unblockApplicationPage(appId: String) = requiresRole(Role.APIGatekeeper, requiresSuperUser = true) {
+    implicit request => implicit hc => withApp(appId) { app =>
+      Future.successful(Ok(unblock_application(app, isSuperUser, unblockApplicationForm.fill(UnblockApplicationForm("")))))
```

Comment:
This page seems to require you to be a superuser but shows some content conditionally based on whether you're a super user?